src/stockwatch/use_cases/degiro.py

```diff
@@ -2,14 +2,22 @@
 import threading
 from dataclasses import dataclass
 from datetime import date, timedelta
+from http import HTTPStatus
 
 import requests
+from requests.api import head
```

Comment:
this line can be removed and causes pylint to fail

Reply:
Ahh indeed, some auto addition added by my editor. Wasn't captured locally as for me apparently just calling `tox` wasn't actually running the style and lint checks.